node/parachain/Cargo.toml

```diff
@@ -62,12 +62,12 @@ pallet-transaction-payment-rpc = { git = "https://github.com/paritytech/substrat
 sp-block-builder = { git = "https://github.com/paritytech/substrate", branch = "master" }
 sc-consensus-manual-seal = { git = "https://github.com/paritytech/substrate", branch = "master" }
 
-evm = { package = "pallet-evm", git = "https://github.com/purestake/frontier", branch = "v0.4-hotfixes" }
-ethereum = { package = "pallet-ethereum", git = "https://github.com/purestake/frontier", branch = "v0.4-hotfixes" }
+evm = { package = "pallet-evm", git = "https://github.com/purestake/frontier", branch = "v0.5-hotfixes" }
+ethereum = { package = "pallet-ethereum", git = "https://github.com/purestake/frontier", branch = "v0.5-hotfixes" }
 
-frontier-rpc = { package = "fc-rpc", git = "https://github.com/purestake/frontier", branch = "v0.4-hotfixes" }
-frontier-rpc-primitives = { package = "fp-rpc", git = "https://github.com/purestake/frontier", branch = "v0.4-hotfixes" }
-frontier-consensus = { package = "fc-consensus", git = "https://github.com/purestake/frontier", branch = "v0.4-hotfixes" }
+frontier-rpc = { package = "fc-rpc", git = "https://github.com/purestake/frontier", branch = "v0.5-hotfixes" }
```

Comment:
Why are frontier packages renamed?

Reply:
They've been this way for a little while. It started because the crates first got re-named upstream. I aliased them in the moonbeam repo as part of https://github.com/PureStake/moonbeam/pull/102 in order to not make that PR any bigger than it already was. This is already cleaned up on master (I did that in https://github.com/PureStake/moonbeam/pull/163) and those changes will need to be merged in here.

Reply:
I'm all for not renaming them. That just adds complexity.